Makefile

```diff
@@ -1 +1,3 @@
 -include node_modules/marketplace-gulp/Makefile
+
+TEST_URL?='http://localhost:8674'
```

Comment:
You have to use `shell`.

```
PORT?=$(shell grep 'PORT: \d\d*' config.js | grep -o '\d\d*')
TEST_URL?='http://localhost:${PORT}'
```

From https://www.gnu.org/software/make/manual/make.html#Shell-Function.